src/main/java/com/amadeus/Response.java

```diff
@@ -121,6 +121,11 @@ private void parseData(HTTPClient client) {
 
   // Tries to read the body.
   private String readBody() {
+    // Workaround to avoid ParserException when status code is 204
+    if (statusCode == 204) {
+      body = "{ \"data\": { } } ";
```

Comment:
Does it mean you want the SDK to return a body "{ \"data\": { } } " when doing a delete?